_posts/2023-03-13-introducing-security-analytics.md

```diff
@@ -0,0 +1,72 @@
+
+Security Analytics takes a different approach. It lets businesses use their existing OpenSearch cluster for detecting security threats, rather than requiring them to bear the cost of duplicating and storing their data for a separate off-the-shelf security solution. Security Analytics also leverages a schema of rules that it applies to log data to look for matches with abnormal events. These standardized rules address specific threats known throughout the industry and form a knowledge base of known attacks. Combined with a number of robust features, the rules also allow Security Analytics to be highly customizable.
+
+## The parts of the whole
+
+To see how this works, let’s first look at some of the components in Security Analytics.
+
+### Detectors
+
+Central to Security Analytics are its detectors. Detectors are components that interpret log data, compare that data to a schema of rules, and define the triggers that generate findings and the alerts that notify administrators of possible security events in the system. Their advantage is that they offer flexibility through their configuration, and this allows a business to fine tune its detection capabilities for specific threats, wider objectives, or targets in between.
+
+### Rules
+
+Security Analytics leverages the versatility, portability, and up-to-date framework of [Sigma rules](https://github.com/SigmaHQ/sigma). These rules define the conditional logic applied to ingested log data and allow the system to identify an event of interest. In general, they determine exactly what type of event or attack the system is looking out for. Security Analytics uses these prepackaged, open-source rules as a starting point for describing relevant log events. But with their inherently flexible format and wide compatibility, users can import and customize Sigma rules to meet their needs.
+
+### Log types
+
+Log data is key to understanding activity in any network. Whether it’s server logs, firewall logs, infrastructure logs, application logs, or endpoint logs, this data is elemental in gaining insight into the activity within a network. In terms of security coverage, the more log data a security infrastructure can process the better. Similarly, the wider the variety of log types that infrastructure can ingest, the more versatile that solution. Security Analytics launched with options for the most common log types. Since then, we have devoted significant energy to expanding support for more and more log types. And that trend will continue.
```

Comment:
i would remove phrases like "more and more" and replace it with "more" or "additional"